### ISSUE TITLE
New Character Creation

### ISSUE THREAD

**Issue by AussieOfficer** [renzuzu/renzu_multicharacter#71]
Hi i have been having a issue when you create a new character it just doesn't load the character outfit section and i have looked through the codes and cant find the issue its pertaining to 

[script:renzu_multich] SCRIPT ERROR: citizen:/scripting/lua/scheduler.lua:739: SCRIPT ERROR: @qb-core/server/player.lua:166: attempt to index a nil value (field '?')
[script:renzu_multich] > CheckPlayerData (@qb-core/server/player.lua:166)
[script:renzu_multich] > ref (@qb-core/server/player.lua:30)
[script:renzu_multich] > rawExecute (@oxmysql/dist/build.js:25254)
[script:renzu_multich] > processTicksAndRejections (node:internal/process/task_queues:96)
[script:renzu_multich] 
[script:renzu_multich] > Login (@renzu_multicharacter/server/framework/main.lua:145)
[script:renzu_multich] > ? (@renzu_multicharacter/server/main.lua:23)
[script:renzu_multich] > handler (@renzu_multicharacter/server/main.lua:9)
[script:renzu_multich] > rawExecute (@oxmysql/dist/build.js:25254)
[script:renzu_multich] > processTicksAndRejections (node:internal/process/task_queues:96)

**Comment by 2Wavyy** [comment]
Yes I'm having this same issue. The either spawn in without menu or spawn in dead
